src/test/scala/dxWDL/util/InstanceTypeDBTest.scala

```diff
@@ -415,4 +415,15 @@ class InstanceTypeDBTest extends FlatSpec with Matchers {
       db.chooseAttrs(None, None, Some(8), Some(false))
     }
   }
+
+  it should "Query returns correct pricing models for org and user" in {
```

Comment:
@orodeh @mhrvol  Doesn't this test depend on who’s running it (who has specific permissions to get pricing)? When the unit tests are run locally by different users it may fail or not. I suggest either i. removing the test and brining it back later with a conditional testing (skipping if) when we switch to GHA and update the Travis token or ii. adding a check if the user can get actually the prices and then assert (I am against option ii since it's testing something that is already tested on the API server side).